_inc/client/plans/single-product-backup.scss

```diff
@@ -1,3 +1,21 @@
+@import '../scss/variables/_colors.scss';
+
+.plans-section__header {
+	color: #23282d;
+	text-align: center;
+	font-family: Roboto, sans-serif;
+	font-weight: normal;
+	font-size: 22pt;
+}
+
+.plans-section__subheader {
+	color: #747474;
+	text-align: center;
+	font-family: Roboto, sans-serif;
```

Comment:
The same as in the other comment - I think we shouldn't override the font family here and leave a default font.